velox/common/caching/SsdFile.h

```diff
@@ -297,6 +298,7 @@ class SsdFile {
   };
 
   static constexpr uint64_t kRegionSize = 1 << 26; // 64MB
+  static constexpr int32_t kBufferFlushThreshold = 1 << 20; // 1MB
```

Comment:
I name it as 'threshold' because to keep the current logic simple, currently we flush when the buffered data size >= `threshold`, meaning that at some point, the buffered data size may be larger than `threshold`.

vs `kCheckpointBufferSize`: It should be a hard limit on the size. The buffered data cannot be larger than the size at any point.